src/test/java/net/snowflake/client/jdbc/ResultSetIT.java

```diff
@@ -1143,6 +1124,34 @@ public void testMultipleChunks() throws SQLException, IOException {
     telemetry.sendBatchAsync();
   }
 
+  @Test
```

Comment:
@sfc-gh-azhan I have added a test case that asserts that the telemetry log is present and has the correct information in it after one of the DatabaseMetadata get functions is called.

Reply:
See my comments above